src/ui/components/shared/SharingModal.js

```diff
@@ -171,12 +170,16 @@ function EmailForm({ recordingId }) {
   if (status.type === "fetched-user") {
     if (status.error) {
       return <ErrorHandler message={"We can not fetch that collaborator right now."} />;
-    } else if (status.data.users.length === 0) {
+    } else if (status.data.user_id_by_email.length === 0) {
```

Comment:
As a follow up, i'm not thrilled with how the graphql data structures leak into our UI. It would be nice to have tighter formatting closer to the queries so that we can normalize this data a bit more.

So for instance, by the time we're in this component it would be nice to expect a user record that had an ID.